packages/openapi3/src/schema-emitter.ts

```diff
@@ -157,6 +163,7 @@ export class OpenAPI3SchemaEmitterBase<
 
   modelDeclaration(model: Model, _: string): EmitterOutput<object> {
     const program = this.emitter.getProgram();
+    validateComponentFixedFieldKey(program, model, model.name);
```

Comment:
because it is now a warning it should still provude valid openapi spec so the name needs to be patched to remove the disallowed values.

Reply:
Plan to implement:

collect all invalid name in global state map.
remove all invalid names when OpenAPI document is ready
Do you think it right?

Reply:
No don't drop things, just remove the invalid chars and replace with either nothing or some placeholder

Reply:
oh.... typo, names --> characters